src/doubleTrouble.js

```diff
@@ -24,7 +24,17 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(numbers, target) {
-  // write code here
+  let findedIndex = numbers.findIndex((item, index, arr) =>
+    item + arr[index - 1] === target);
+  let resultArr = numbers.filter((item, index) => index !== findedIndex);
+
+  while (findedIndex !== -1) {
+    findedIndex = resultArr.findIndex((item, index, arr) =>
+      item + arr[index - 1] === target);
+    resultArr = resultArr.filter((item, index) => index !== findedIndex);
```

Comment:
you repeat the same code

Reply:
> you repeat the same code

this isn't the same - the first i work with "native" array - from the function's argument and store the result in new array and then in the cycle i work with new array an return it. But, no problem i can create new array at once and worked with is only/
Refactored